render/naga-pixelbender/src/lib.rs

```diff
@@ -410,9 +410,26 @@ impl ShaderBuilder<'_> {
         // We've emitted all of the opcodes into the function body, so we can now load
         // from the destination register and return it from the function.
         let dst_load = builder.load_src_register(dst)?;
-        builder.push_statement(Statement::Return {
-            value: Some(dst_load),
-        });
+
+        // Set the alpha channel to 1 if the number of channels is <4.
```

Comment:
I _think_ there's an easier way than manually composing R/G/B/1; a direct translation of `value.a = 1.0` would be something like:
```rs
            Statement::Store {
                pointer: builder.evaluate_expr(Expression::AccessIndex { base: dst_load, index: 3 })
                value: builder.onef32,
            },
```
I didn't actually test this though.